.github/workflows/tests.yml

```diff
@@ -97,6 +97,26 @@ jobs:
           directory: ./coverage
           flags: front,unit_front
 
+  build:
+    name: 'build (node: ${{ matrix.node }})'
+    needs: [lint, unit_front]
```

Comment:
Should this only depend on the `lint` job?

Reply:
i thought about this, but my pov was "no point building if your tests dont pass", wdyt?